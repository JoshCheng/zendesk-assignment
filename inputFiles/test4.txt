3 4
Enter motorcycle SGX1234A 1613541902
Enter car SGF9283P 1613541902
Exit SGX1234A 1613545602
Enter car SGP2937F 1613546029
Exit SGF9283P 1613542902
Exit SGF9283P 1613541111
Enter motorcycle ABC1234A 16111
Enter motorcycle ABC1234A 16111
Enter car SDW2111W 1613549730
Enter car SDW2111W 1613549730
Enter car SSD9281L 1613549740
Enter car SSD9281C 1613549741
Exit SDW2111W 1613559745
Enter car SSD9281C 1613549741