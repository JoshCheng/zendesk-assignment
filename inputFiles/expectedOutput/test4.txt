Accept MotorcycleLot1
Accept CarLot1
MotorcycleLot1 2
Accept CarLot2
CarLot1 1
Error occurred at line 7 - Vehicle number SGF9283P cannot be found within parking lots.
Accept MotorcycleLot1
Accept MotorcycleLot2
Accept CarLot1
Accept CarLot3
Reject
Reject
CarLot1 6
Accept CarLot1
